### DISCUSSION TITLE
家人们，Win11系统突然用不了了啥情况，Chrome打不开，IP更新点击后会闪退，拜托了

### DISCUSSION THREAD

**Discussion by X1203xq** [bannedbook/fanqiang#875] [Q&A]
![1](https://github.com/bannedbook/fanqiang/assets/162563963/d18ac1f3-f0fa-4256-83cd-1c0cca866c02)
![2](https://github.com/bannedbook/fanqiang/assets/162563963/2853fcb8-958a-41a4-9cc7-125261d042f2)
![1](https://github.com/bannedbook/fanqiang/assets/162563963/3b829c53-ece3-4e9d-92ae-2f9a628b665b)
ge

**Comment by X1203xq** [comment]
![Uploading 3.png…]()

**Comment by X1203xq** [comment]
![4](https://github.com/bannedbook/fanqiang/assets/162563963/40a81e74-285f-4678-b897-ca23a5df2ea1)
换个浏览器也不行

**Comment by X1203xq** [comment]
感谢感谢～

---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年03月07日 17:44 |
| 收件人 | bannedbook/fanqiang ***@***.***> |
| 抄送人 | X1203xq ***@***.***>,
Author ***@***.***> |
| 主题 | Re: [bannedbook/fanqiang] 家人们，Win11系统突然用不了了啥情况，Chrome打不开，IP更新点击后会闪退，拜托了 (Discussion #875) |

晚点研究下，可能需要更新程序

**Comment by bannedbook** [comment]
你这个windows 11 ，之前可以正常使用吗
